src/modules/infobox/components/FeatureProfile.vue

```diff
@@ -218,27 +230,35 @@ export default {
             }
         },
         async createProfileChart() {
-            const data = await this.getProfile(this.featureCoordinates)
+            const data = await this.getProfile()
             this.profileChart.create(data)
             const areaChartPath = this.profileChart.group.select('.profile-area')
             const glass = this.profileChart.glass
             this.attachPathListeners(areaChartPath, glass)
             return this.profileChart.element
         },
         async updateProfileChart(size) {
-            const data = await this.getProfile(this.featureCoordinates)
+            const data = await this.getProfile()
             this.profileChart.update(data, size)
             return this.profileChart.element
         },
-        async getProfile(coordinates) {
+        async getProfile(apiFunction = profileJson, coordinates = this.featureCoordinates) {
             const coordinatesLv95 = toLv95(coordinates, 'EPSG:3857')
-            return profile({
-                geom: `{"type":"LineString","coordinates":${JSON.stringify(coordinatesLv95)}}`,
+            return apiFunction({
+                geom: { type: 'LineString', coordinates: coordinatesLv95 },
                 offset: 0,
                 sr: 2056,
                 distinct_points: true,
             })
         },
+        triggerDownload(blob, fileName) {
+            const link = document.createElement('a')
+            link.href = URL.createObjectURL(blob)
+            link.download = fileName
+
+            link.click()
+            URL.revokeObjectURL(link.href)
+        },
```

Comment:
could you add a little comment that we must use this `<a>` approach so that we can define the downloaded file name?
otherwise the next one to come here will want to switch to a `window.open(...)` 😉